packages/rollups-node/Dockerfile

```diff
@@ -8,6 +8,7 @@ FROM ${REGISTRY}/${ORG}/rollups-dispatcher:${ROLLUPS_VERSION}      AS dispatcher
 FROM ${REGISTRY}/${ORG}/rollups-advance-runner:${ROLLUPS_VERSION}  AS advance_runner
 FROM ${REGISTRY}/${ORG}/rollups-inspect-server:${ROLLUPS_VERSION}  AS inspect_server
 FROM ${REGISTRY}/${ORG}/rollups-graphql-server:${ROLLUPS_VERSION}  AS graphql_server
+FROM ${REGISTRY}/${ORG}/rollups-host-runner:1.0.0                  AS host-runner
```

Comment:
```suggestion
FROM ${REGISTRY}/${ORG}/rollups-host-runner:${ROLLUPS_VERSION}     AS host-runner
```

Reply:
The `rollups-host-runner` docker image  versions is not as same as other services : 

https://hub.docker.com/r/cartesi/rollups-host-runner/tags

Reply:
rollups-host-runner` was just released at 1.0.0, previously it was on a different repository and name https://github.com/cartesi/host-server-manager
 
 reusing `ROLLUPS_VERSION` for rollups-host-runner is safe here, we could even consider renaming it to `ROLLUPS_NODE_VERSION`, to be more precise

Reply:
Fixed !